### ISSUE TITLE
Feature request: "per_task" type of CSV report

### ISSUE THREAD

**Issue by pwa-tapptic** [larose/utt#100]
Hello guys, thanks for the awesome tool you've created! 
I'm rather hardcore UTT user and it would be extremely helpful to extend a bit the CSV reports, for they can be nicely "piped" into tools such as `csvlens` (`utt ... --csv-section ... | csvlens`). Now there is actually only `per_day` "mode" which is not entirely helpful in analyzing a particular day(s) in details. 

The proposition would be to have `per_task` that will print a bit detailed overview, i.e.:
```csv
Tasks,Project,Date,Duration,
"coding","awesome project",2023-10-24,1h55
"meeting","awesome project",2023-10-24,1h55
"reviewing","awesome project",2023-10-24,1h55
```

`csvlens` print:
```
────────────────────────────────────────────────────────────────────────────────
      Tasks        Project            Date          Duration
───┬────────────────────────────────────────────────────────────┬───────────────
1  │  coding       awesome project    2023-10-24    1h55        │
2  │  meeting      awesome project    2023-10-24    1h55        │
3  │  reviewing    awesome project    2023-10-24    1h55        │
```

I could help in contributing if you find it worth including in the main package.

**Comment by pwa-tapptic** [comment]
@larose what do you think? I can implement it as my local extension but IMO it makes sense to include it in the main package, for idea of different types of csv reports is actually expressed in the parameters.

**Comment by larose** [comment]
Hi @pwa-tapptic! Absolutely, I'd appreciate your contribution as it appears to be a valid use case. To include it in this repo, make sure your pull request has at least some integration tests. Thanks!

**Comment by larose** [comment]
Implemented in #101
